### ISSUE TITLE
[0.9.0 release-86] Waste filter improvement:

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#18583]
Save file:
https://drive.google.com/file/d/15pHazQFpTwFa7bvngsNP2u-FMj1sECyu/view?usp=sharing
- [ ] 1. Don't have any error in status when you connect waste filter 1 output to waste filter 2 input. All craft process is stopped but Waste filter have green status:
![image](https://user-images.githubusercontent.com/45708377/95059635-d41d8c00-0701-11eb-8ea3-798b6400cbf7.png)
![image](https://user-images.githubusercontent.com/45708377/95059763-0333fd80-0702-11eb-88ec-67c510b34cda.png)
Coordinates: 680,60,580.

- [ ] 2. Waste Filter doesn't start to work from Combustion generator if Combustion generator is only one object that give electric power. But After my investigate I guess that  Waste Filter can work without electric power (Seems it still have something like battery) But after time it stops.
![image](https://user-images.githubusercontent.com/45708377/95067575-b73a8600-070c-11eb-84fa-f04642b820b5.png)
Coordinates: 220,60,560.

**Comment by SlayksWood** [comment]
> **Version:** 0.9.0.5 beta release-100
> 
> **Expected behavior:**
> Waste Filter is processing Sewage, producing Compost, but does not consume power.
> 
> **Actual behavior:**
> Waste Filter consumes power when working.
> 
> **Do you have mods installed? Does the issue happen when no mods are installed?:**
> no

**Comment by SlayksWood** [comment]
9.1.7 release-126
> I placed a waste filter, everything is connected properly, there's power, it's on a claim, and there's a stockpile next to it.
> But the waste filter does not consume ANY power, it always shows as "0 watts"
> 
> Also, if you put a bend pipe on the output of the waste filter, it generates nothing as output, but it clears the input waste (infinite free waste blackhole?)
> 
> Power :
> ![image](https://user-images.githubusercontent.com/4887819/100391038-8bfc5500-3032-11eb-94e2-8b1b6d46016d.png)
> 
> Flowing output :
> ![image](https://user-images.githubusercontent.com/4887819/100391052-961e5380-3032-11eb-9006-e9cc453a2735.png)
> 
> Sewage input :
> ![image](https://user-images.githubusercontent.com/4887819/100391061-9e768e80-3032-11eb-896a-72235e892e7c.png)
> 
> Water output :
> ![image](https://user-images.githubusercontent.com/4887819/100391067-a3d3d900-3032-11eb-80fc-68aef3ee7985.png)
> 
> Status :
> ![image](https://user-images.githubusercontent.com/4887819/100391075-a9312380-3032-11eb-8792-562b86dc08b0.png)
> 
> Game version :
> ![image](https://user-images.githubusercontent.com/4887819/100391174-eb5a6500-3032-11eb-81e4-69bbe5c51413.png)

> Same problem here:
> Working, enabled and active but consuming 0W
> ![image](https://user-images.githubusercontent.com/22171518/100391084-ad5d4100-3032-11eb-933b-8903180ac6b6.png)
> ![image](https://user-images.githubusercontent.com/22171518/100391089-b0f0c800-3032-11eb-815f-9bc89cc802fe.png)
> ![image](https://user-images.githubusercontent.com/22171518/100391093-b51ce580-3032-11eb-836f-bdd3551f7b68.png)
> Full setup:
> ![image](https://user-images.githubusercontent.com/22171518/100391116-c8c84c00-3032-11eb-8e17-21c1e562e1ae.png)
> 
> Version 0.9.1.7 release-126

**Comment by F0x06** [comment]
Apparently, the machine stops generating compost when you connect multiple sewage outputs to it, it works with a single sweage output, i still see 0w usage in both scenarios.

**Comment by IcedForge** [comment]
Yeah summary is It needs an active power source to work. But its not occupying the required amount of power.
If you disable the power generating source it will stop all machines from producing sewage with blocking them as per no sewage output possible.
